week1/3-website/index.html

```diff
@@ -1,22 +1,113 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8" />
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no" />
     <title>My Blog</title>
-    <link
-      href="//fonts.googleapis.com/css?family=Roboto:400,500,300"
-      rel="stylesheet"
-      type="text/css"
-    />
+    <link href="//fonts.googleapis.com/css?family=Roboto:400,500,300" rel="stylesheet" type="text/css" />
     <link rel="stylesheet" href="css/style.css" />
     <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+    <link rel="stylesheet" type="text/css" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.css">
   </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
-  </body>
-</html>
+  <main class="container">
+    <header>
+      <h1 class="topic"> My Coding Brag Blog </h1>
+      <menu class="nav">
+        <nav id="navigation">
+          <a class="link" href="#Intro"> Pre-Bootcamp </a>
+          <a class="link" href="#Start"> Drop-in Sessions </a>
+          <a class="link" href="#Fundamentals"> Fundamentals </a>
+          <a class="link" href="#FullStack"> Full Stack Web Dev </a>
+        </nav>
+    </header>
+    <section class="pre">
+      <h2>Tech Passion and Quest</h2>
+      <p> <img class="profile-image" src="https://pbs.twimg.com/profile_images/1218723944443891712/2KB9GnZv_400x400.jpg"
+        alt="Profile picture"> Welcome to my first blog ever! Yay!!! Well, I have always had a passion for technology.
+        My quest for developing my skills prompted me to attend a few paid trainings. But with no mentor, no
+        proper structure of learning, no road map for personal development, my focus always
+        shifted away for a couple of reasons including lack of direction and no support. My passion meant I
+        followed a few trainers on YouTube but I self learning is always going to be a challenge.
+        But then I was pretty fortunate to have my wife's friend, a Software Developer, introduced me to Code Your
+        Furure Bootcamp. <br>
+        So, below is a summary of the study path so far and the journey ahead. Hope you have fun!!!
+      </p>  
+      <div class="social">
+        <ul>
+          <li> <i class="fa fa-facebook"></i> </li>
+          <li> <i class="fa fa fa-twitter"></i> </li>
```

Comment:
Applied the 'fa' class twice

Reply:
Corrected it. I have made  I believe you meant only the one on facebook link. I initially tried fa fa (twice) for all of them and then fa fa fa (thrice). I got same result. Not quite sure why???